### ISSUE TITLE
Refactor the project

### ISSUE THREAD

**Issue by iagocavalcante** [zhukov/webogram#1751]
I would like to help refactoring some pieces of code and send to you PR. I started with the controllers, separate each controller in one file, give more context and with this i know each controller is responsible for one thing.

You can see this changes in my forked repository and if you feel confortable i can make the PR.

https://github.com/iagocavalcante/webogram/commits/master

**Comment by erezson** [comment]
I have the feeling that they're going to have a complete new web product after the Telegram X

**Comment by iagocavalcante** [comment]
I started this refactoring to understood how it works and improve my skills with angularJS (I'm using this in my current job), I want to build my own client using another tech stack when i finish this refactoring. 

You have more information about Telegram X ?

**Comment by erezson** [comment]
nope, but they just announced that they're going to have Telegram X as the official app in iOS. 
I would assume that they will move to Angular for the web, or something else

**Comment by iagocavalcante** [comment]
So, I don't will send the pull request. Thanks Bro!

**Comment by stek29** [comment]
@erezson How is that feeling justified? Also, webogram is already built using Angular.

**Comment by iagocavalcante** [comment]
I think, I can't send the PR now, because this is a huge change in the architecture. But I really joy if you help me with code review @erezson and @stek29

**Comment by erezson** [comment]
@iagocavalcante sorry, I'm not a developer but Product Manager :)

@stek29 I'm not saying that this is what gonna happen, but companies are moving from angularjs to angular so I would suggest to ask Telegram if there is a reason to have now refactoring before he starts

**Comment by azzimuth** [comment]
@stek29, don't mix up Angular and AngularJS ;)
